WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginNotification.java

```diff
@@ -0,0 +1,69 @@
+package org.wordpress.android.ui.accounts.login;
+
+import android.app.Notification;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.BitmapFactory;
+import android.support.annotation.StringRes;
+import android.support.v4.app.NotificationCompat;
+
+import org.wordpress.android.R;
+import org.wordpress.android.util.AutoForeground;
+
+class LoginNotification {
+    private static Intent getPendingIntent(Context context) {
+        Intent resumeIntent = context.getPackageManager().getLaunchIntentForPackage(context.getPackageName());
+
+        // apparently, we need to null out the package name to make the intent resume the app
+        resumeIntent.setSelector(null);
+        resumeIntent.setPackage(null);
+
+        return resumeIntent;
+    }
+
+    private static NotificationCompat.Builder getNotificationBuilder(Context context, @StringRes int title,
+            @StringRes int content) {
+        NotificationCompat.BigTextStyle bigTextStyle = new NotificationCompat.BigTextStyle();
```

Comment:
I think we need to set the normal notification text as well here, otherwise when the notification is collapsed a blank notification is shown:

<img width="439" alt="screen shot 2017-11-17 at 9 10 54 pm" src="https://user-images.githubusercontent.com/6597771/32974317-d87d8e6c-cbdb-11e7-8d4b-b88e79a73097.png">

Once you drag the bottom edge down to expand it, the bigstyle text appears:

<img width="441" alt="screen shot 2017-11-17 at 9 11 25 pm" src="https://user-images.githubusercontent.com/6597771/32974340-f815a46c-cbdb-11e7-96a7-51b3c90fa4d6.png">

So, we'd need to add
```
        notificationBuilder.setContentTitle(title);
        notificationBuilder.setContentText(content);
```
to the `NotificationCompat.Builder(context)` concatenation right below

Reply:
Good call. I opted (with 01b1449) to setting the standard title only so the collapsed notification can look, well, collapsed :). Let me know if this is OK or you'd prefer the standard contentText to be set as well. Thanks!

In the meantime, I removed the big icon anyway to match other notification styles in the app and this has the added benefit that the collapsed notification looks less empty.

Reply:
I think I would rather have both the title and the text on the notification so the user can know the reasons why "Login stopped" and which actions are required right at a glance. Otherwise they'd only see the title i..e "Login stopped" with no further reasons or actions to take (the "Please provide an authentication code to continue" is fully informative in this regard, you know what's about to happen after tapping, as opposed to not knowing what to expect with just the  "login stopped" title).

Reply:
Hmm, I see your point @mzorz . I think that the standard content text doesn't allow for multiple lines though and the informative texts you mention actually get clipped. It depends on the screen size too of course but, that's basically the reason I employed the BigStyle notification as well.

I made the change with 398faac.

Reply:
I know, it's tricky to have all possibilities in mind. From my experience, best way to keep it with high chances of showing for everybody  / giving people chance to just expand without opening to try and see what happened, is to set title and content on both normal and BigStyle, as it is now 👍